src/coffea/util.py

```diff
@@ -195,3 +197,15 @@ def rewrap_recordarray(layout, depth, data):
     if isinstance(layout, awkward.layout.RecordArray):
         return lambda: data
     return None
+
+
+# shorthand for compressing forms
+def compress_form(formjson):
+    return binascii.b2a_base64(
```

Comment:
I have a personal preference for `base64.base64encode` but I can't see any real difference between them https://stackoverflow.com/questions/20927325/why-is-there-a-difference-between-binascii-b2a-base64-and-base64-b64encode